src/lib.rs

```diff
@@ -51,6 +50,13 @@ pub extern "C" fn transform(input: *const libc::c_char) -> *const libc::c_char {
     return c_str.into_raw();
 }
 
+#[no_mangle]
+pub extern "C" fn noop(input: *const libc::c_char) -> *const libc::c_char {
```

Comment:
I think this is the same as `passthrough` so you can remove that function.

Reply:
same with line 187 on `main.go`

Reply:
👍